resources/settings.xml

```diff
@@ -1,189 +1,244 @@
-		<setting id="apikey.tadb" label="32969" type="text" />
-		<!-- For add-on inner workings -->
-		<setting id="last_videoupdate" type="text" default="0" visible="false" />
-		<setting id="last_musicupdate" type="text" default="0" visible="false" />
-		<setting id="check_allepisodes" type="bool" default="true" visible="false" />
-		<setting id="autoaddepisodes_list" type="text" default="" visible="false" />
-	</category>
+        <setting label="32973" type="lsep" />
+        <setting id="language_override" label="32913" type="select" default="None" values="None|en|fr|de|ja|zh|es|it|pt|sv|ru|nl|ar|ko|no|hu|da|hi|is|pl|he|bg|fi|ml" />
+        <setting id="language_fallback_kodi" label="32974" type="bool" default="true" />
+        <setting id="language_fallback_en" label="32975" type="bool" default="true" />
+        <setting label="$LOCALIZE[744]" type="lsep" />
+        <setting id="episode.thumb_generate" label="32914" type="bool" default="false" />
+        <setting id="movie.thumb_generate" label="32915" type="bool" default="false" />
+        <setting id="musicvideo.thumb_generate" label="32920" type="bool" default="false" />
+        <setting id="centraldir.set_enabled" label="32909" type="bool" default="false" />
+        <setting id="centraldir.set_dir" subsetting="true" label="32910" type="folder" visible="eq(-1,true)" />
+        <setting id="setartwork_subdirs" subsetting="true" label="32976" type="bool" default="true" visible="eq(-2,true)" />
+        <setting id="setartwork_fromparent" label="32911" type="bool" default="false" visible="eq(-3,false)" />
+        <setting id="albumartwithmediafiles" label="32951" type="bool" default="false" />
+        <setting id="savewith_basefilename" label="32941" type="bool" default="true" />
+        <setting id="savewith_basefilename_mvids" label="32942" type="bool" default="true" />
+        <setting id="save_extrafanart" label="32929" type="bool" default="false" />
+        <setting id="save_extrafanart_mvids" label="32930" type="bool" default="false" />
+        <setting id="identify_alternatives" label="32906" type="bool" default="true" />
+        <setting label="32965" type="lsep" />
+        <setting id="apikey.fanarttv" label="32966" type="text" />
+        <setting id="apikey.tvdb" label="32967" type="text" />
+        <setting id="apikey.tmdb" label="32968" type="text" />
+        <setting id="apikey.tadb" label="32969" type="text" />
+
+        <!-- file path exclusion options -->
+        <setting label="32820" type="lsep" />
+        <setting id="exclude.path.option_1" type="bool" label="32821" default="false" />
+        <setting id="exclude.path.type_1" type="select" label="32822" lvalues="32823|32824|32825" visible="eq(-1,true)" />
+        <setting id="exclude.path.folder_1" type="folder" label="32826" default="" visible="eq(-2,true)+eq(-1,0)" source="" />
+        <setting id="exclude.path.text_1" type="text" label="32827" default="" visible="eq(-3,true)+eq(-2,1)" />
```

Comment:
Can you rename this `text` option to **prefix** throughout the code? It's purpose will be more obvious looking at it fresh in the future.